### DISCUSSION TITLE
What is the replacement for pimcore:generate:bundle?

### DISCUSSION THREAD

**Discussion by Blackbam** [pimcore/pimcore#10002] [General]
In Pimcore 6.x we were using pimcore:generate:bundle for generating bundles. This command has been removed? Bundles are now stored in /bundles, but still a generator for bundle skeletons would be useful right? What is the replacement?

**Comment by dvesh3** [comment]
@Blackbam Hi, use this bundle https://github.com/pimcore/bundle-generator

**Reply by Blackbam** [reply]
Thanks!
